healthcare/dicomweb_study_search.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START healthcare_search_studies]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	healthcare "google.golang.org/api/healthcare/v1beta1"
+)
+
+// Create a struct to append the query parameter because the library does not currently
+// natively support this.
+type queryParam struct {
```

Comment:
Name this `queryParamOpt` or `queryParamOption`?

Reply:
Renamed to queryParamOpt.